test/vms/ramen/install

```diff
@@ -6,5 +6,7 @@
 chdir $(dirname $0)
 cp usr/bin/ramen /usr/bin
 cp etc/init.d/ramen /etc/init.d
+cp etc/init.d/increase-root-disk-size /etc/init.d
+ln -sn ../init.d/increase-root-filesystem-size /etc/rc3.d/S97-increase-root-filesystem-size
```

Comment:
This is called `growfs` in cloud-init. Since this basically does resize-filesystem and we have only one filessystem, we can use a shorter name like `resize-filesystem`.

Reply:
done